packages/components/CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 #### Changed
 
+- `visibility` css property can be overridden on all components, e.g. to make use of `visibility: hidden;`  
```

Comment:
Even tho we have an issue with releasing atm, this should go up in the unreleased section.

Reply:
bad auto merge